roles/confluent.control_center/tasks/health_check.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Wait for webpage to serve HTTP content
```

Comment:
in a future PR could probably combine these two tasks into one

Reply:
I would suggest filing jiras for this, as opposed to leaving comments on the repo.  Once we merge, this will get lost.